service/resource/deployment/resource.go

```diff
@@ -85,6 +84,9 @@ func New(config Config) (*Resource, error) {
 	if config.Logger == nil {
 		return nil, microerror.Maskf(invalidConfigError, "config.Logger must not be empty.")
 	}
+	if config.TemplateURIVersion == "" {
+		return nil, microerror.Maskf(invalidConfigError, "config.TemplateURIVersion must not be empty.")
```

Comment:
Super unimportant nitpick. These are not really sentences. So the full stop at the end of error message is wrong IMO. We never did it that way.

Reply:
copy/paste. I'll fix that.

Reply:
Done.